libs/application/templates/estate/src/fields/EstateMembersRepeater/index.tsx

```
@@ -204,29 +203,78 @@ export const EstateMembersRepeater: FC<
+                      key={`${id}[${index}].dateOfBirth`}
+                      id={`${id}[${index}].dateOfBirth`}
+                      name={`${id}[${index}].dateOfBirth`}
+                      locale="is"
+                      maxDate={new Date()}
+                      minYear={1900}
+                      maxYear={new Date().getFullYear()}
+                      backgroundColor="blue"
+                      onChange={(d) => {
+                        setValue(`${id}[${index}].dateOfBirth`, d)
+                      }}
+                      error={error && error[index] && error[index].dateOfBirth}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)